### ISSUE TITLE
When running Wallet only mode, unable to run Split coins command[Bug] 

### ISSUE THREAD

**Issue by shaneo257** [Chia-Network/chia-blockchain#15527]
### What happened?

When trying to run the `Split` coins command when running in Wallet only mode, it throws an error that the wallet needs to be in sync, when switching to farming mode, it works as intended, reported by user in discord https://discord.com/channels/1034523881404370984/1034870571864948777/1118370242066923652

### Version

1.8.1

### What platform are you using?

Linux

### What ui mode are you using?

GUI

### Relevant log output

```shell
chia wallet coins split -n 1 -a .01 -t 0x2f663a3298190844cfef29c86a0c82e096a72bcec3cc1e90fe705d1d13351e1f  Exception from 'wallet' {'error': 'Wallet needs to be fully synced before sending transactions', 'success': False}:
Traceback (most recent call last):
  File "chia/cmds/cmds_util.py", line 116, in get_any_service_client
  File "chia/cmds/cmds_util.py", line 235, in execute_with_wallet
  File "chia/cmds/coin_funcs.py", line 219, in async_split
  File "chia/rpc/wallet_rpc_client.py", line 214, in send_transaction_multi
  File "chia/rpc/rpc_client.py", line 61, in fetch
ValueError: {'error': 'Wallet needs to be fully synced before sending transactions', 'success': False}
```

**Comment by danieljperry** [comment]
I confirmed the issue with a wallet on which I had been testing clawbacks. That wallet also has a rather lengthy history (maybe 2-3 months) of standard transactions. In order to obtain a clean log file, I created a new wallet, sent it a coin, and tried to split it. This time the split worked.

So the issue is reproduceable with certain wallets, but not others. This testing (both the error and passing cases) was done on 1.8.2rc3, installed on Windows from binary, running the light wallet only.

**Comment by forReason** [comment]
have the same issue on 1.8.1 wallet

**Comment by acoolprogrammer123** [comment]
Still seeing this issue in 1.8.2, running ubuntu in wallet mode on testnet.

**Comment by forReason** [comment]
bump

**Comment by wjblanke** [comment]
We are checking on this. Thanks

**Comment by jack60612** [comment]
the code that raises the error is on line 1033 and 1034, need to look into why its mad when there is no node.
```
        if await self.service.wallet_state_manager.synced() is False:
            raise ValueError("Wallet needs to be fully synced before sending transactions")
```

**Comment by forReason** [comment]
The issue still persists in the 2.0.0 Version on a fresh wallet

**Comment by nick07002** [comment]
Also experiencing this issue.  Please prioritize and fix.

**Comment by sc0-0001** [comment]
This still remains an issue in 2.1.4. Flagging interest in a fix.

**Comment by emlowe** [comment]
We believe this has been fixed in 2.4.4
